src/ArticleTheme.vala

```diff
@@ -14,14 +14,20 @@
 //	along with FeedReader.  If not, see <http://www.gnu.org/licenses/>.
 using Gee;
 
+public struct ThemeInfo {
+    string name;
+    string author;
+    string path;
+}
+
 public class  FeedReader.ArticleTheme {
-    private static ArrayList<HashMap<string, string>> ? themes = null;
+    private static HashMap<string, ThemeInfo?> ? themes = null;
```

Comment:
Why is the second type argument `ThemeInfo?`? Are there cases where we would want them theme to `null`?